### PR TITLE
Issue 4187 - new background image

### LINKED ISSUE #4187: Change background image to one used in Figma
### Description
Background image needs to change for both login and signup pages

Link to background image https://www.figma.com/file/3FjNEOdQbcHwzquVZxUqMl/3D-Repo?node-id=3146%3A24783&t=84kGphFInLknl12U-1

### Pictures/ Gifs of the error

image

### PR DESCRIPTION
**Pull request by Amantini1997**
This fixes #4187 

#### Description
Replaced the background image

### CODE REVIEW

File: frontend/src/v5/ui/components/authTemplate/authTemplate.styles.ts [hunk 1]
```diff
@@ -75,6 +75,8 @@ export const BackgroundOverlay = styled(LoginBackground)`
 	left: 0;
 	right: 0;
 	overflow: hidden;
+	object-fit: cover;
+	object-position: 0 0;
```
Comment (The-Daniel) [inline frontend/src/v5/ui/components/authTemplate/authTemplate.styles.ts:79]: image
On wide screens the background doesn't extend all the way across. I think changing max-width to be min-width and perhaps changing object-fit to "fill" fixes this.

**Comment by george-hadfield** [comment]
Can't see any changes

image

**Comment by Amantini1997** [comment]
> Can't see any changes
> 
> image

There is a an image with some skyscrapers. This is why I see in the figma design at least. Was this not the intended background?

**Review by The-Daniel — APPROVED** [review]

**Comment by george-hadfield** [comment]
The image and graphics should span 100% width of the screen. Where did you see this design in Figma?

**Comment by The-Daniel** [comment]
> The image and graphics should span 100% width of the screen. Where did you see this design in Figma?

He's fixed it so that now it should span the entire width

**Review by george-hadfield — APPROVED** [review]

**Pull request merged**